src/gui/updater/ocupdater.cpp

```diff
@@ -105,14 +105,7 @@ bool OCUpdater::performUpdate()
     if (!updateFile.isEmpty() && QFile(updateFile).exists()
         && !updateSucceeded() /* Someone might have run the updater manually between restarts */) {
         const QString name = Theme::instance()->appNameGUI();
-        if (QMessageBox::information(nullptr, tr("New %1 Update Ready").arg(name),
-                tr("A new update for %1 is about to be installed. The updater may ask\n"
-                   "for additional privileges during the process.")
-                    .arg(name),
-                QMessageBox::Ok)) {
-            slotStartInstaller();
-            return true;
-        }
```

Comment:
Why does is this message box removed there?

Reply:
@FlexW Fixed!